### ISSUE TITLE
Test counterweight motor to tune PID values

### ISSUE THREAD

**Issue by lo9key** [OregonStateRocketry/30k2018-CS-Capstone#6]
- [x] Add counterweight motor to PID loop
- [ ] Tune PID values by hanging the payload from a rope and spinning it, then using counterweight motor to stabilize rotation.

**Comment by lo9key** [comment]
The Payload and ECE teams have not finalized the hardware.

The counterweight motor will be a DC motor, so it needs a PCB take takes a PWM signal and steps up the amperage to the motor. A few days ago we heard they are still working on that.

Leaving this open, but it's unlikely to be finished in Winter term.  We can't progress until the hardware is done.

**Comment by lo9key** [comment]
Just an update to say that we're still waiting on the hardware for this test.

**Comment by lo9key** [comment]
We eventually received the payload about a week before the practice launch, and were able to confirm that the DC motor spun using the payload code.

Before we were able to perform a hanging spin test, the DC motor was damaged during a test of the (unrelated) payload ejection system.

The team tried but were unable to find a replacement in time for the test launch, so the payload was launched without a functional motor.  The GPS system was damaged during launch and the payload continued reporting it's location as the launch pad for ~45 seconds after launch, after which it was never found or heard from again.

Closing because the payload no longer exists and will not be rebuilt.
